SDKs/PowerShell/JumpCloud.SDK.DirectoryInsights/JumpCloud.SDK.DirectoryInsights.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
   <metadata>
     <id>JumpCloud.SDK.DirectoryInsights</id>
-    <version>0.0.33</version>
+    <version>0.0.34</version>
```

Comment:
Looks like we're updating DI version here for some reason

Reply:
These are previous changes from Geoff's branch, not releasing in this PR